packages/shared/src/components/squads/SelectArticle.tsx

```diff
@@ -39,39 +41,50 @@ export function SquadSelectArticle({
     };
   };
   return (
-    <Modal.Body>
-      <SquadTitle>
-        Select one <SquadTitleColor>article.</SquadTitleColor>
-      </SquadTitle>
-      <p className="py-4 text-center">
-        Make your squad aware of your reading history by sharing one article
-        with them
-      </p>
-      <Modal.StepsWrapper>
-        {({ nextStep }) => (
-          <div>
-            {data?.pages.map((page) =>
-              page.readHistory.edges.map((edge) => (
-                <button
-                  key={edge.node.post.id}
-                  type="button"
-                  className="-mx-6 hover:bg-theme-hover cursor-pointer"
-                  onClick={goNext(edge.node, nextStep)}
-                >
-                  <PostItemCard
-                    postItem={edge.node}
-                    showButtons={false}
-                    clickable={false}
-                  />
-                </button>
-              )),
-            )}
-          </div>
+    <>
+      <Modal.Body>
+        <SquadTitle>
+          {!hasData ? 'Read an' : 'Share the first'}{' '}
+          <SquadTitleColor>article</SquadTitleColor>
+        </SquadTitle>
+        <p className="py-4 text-center">
+          {!hasData
+            ? 'Your reading history is empty! Please read at least one article by clicking on a post to start sharing with your Squad'
```

Comment:
Nitpick: I think there should be a `.` at the end since the sentence ends

Reply:
Good point, went for consistency and updated with a dot.